docs/getting_started.rst

```diff
@@ -14,13 +14,23 @@ Install with Conda
 ------------------
 .. _install-with-conda:
 
-`conda <http://www.continuum.io/blog/conda>`_ is a python package manager built for scientific python. Unlike ``easy_install`` or ``pip``, it handles binaries and binary dependencies, which are critical for most scientific workflows. If you're a ``conda`` user, you can install OpenPathSampling by adding the omnia channel. If you're not a conda user, you should look into it. ::
+We recommend using `conda` to install OpenPathSampling.  `conda
+<http://www.continuum.io/blog/conda>`_ is a python package manager built for
+scientific python. Unlike ``easy_install`` or ``pip``, it handles binaries
+and binary dependencies, which are critical for most scientific workflows.
+If you're a ``conda`` user, you can install OpenPathSampling by adding the
+omnia channel. ::
 
-To install OpenPathSampling with conda, use the following commands ::
+To install the most recent release of OpenPathSampling with conda, use the
+following commands ::
 
   $ conda config --add channels http://conda.binstar.org/omnia
```

Comment:
> Is that conda install -c omnia openpathsampling as a replacement for both the commands we had previously?

Yup!

>  I'll update the docs to use that, but want to make sure they're correct (I don't know since I don't use conda, myself).

It's worth testing, and won't impact your existing installation:
```bash
wget --quiet https://repo.continuum.io/miniconda/Miniconda-latest-MacOSX-x86_64.sh
bash Miniconda-latest-MacOSX-x86_64.sh -b -p miniconda
export PATH="$HOME/miniconda/bin:$PATH"
conda install -c omnia openpathsampling-dev --yes
```
Are you sure you don't want to create a non-dev version, even if it's just a preview release?

Also, what's up with no py3.x support?